examples/secretsmanager/secret.yaml

```diff
@@ -1,12 +1,13 @@
 apiVersion: secretsmanager.aws.crossplane.io/v1alpha1
 kind: Secret
 metadata:
-  name: example-secret
+  name: example-secret-3
 spec:
   forProvider:
     region: us-east-1
-    description: "secrets of test cluster created with writeConnectionSecretToRef"
-    kmsKeyIdRef: example-key-id
+    description: "test"
+#    kmsKeyIDRef:
+#      name: example-key-id
```

Comment:
looks like we haven't included this in a release yet so no need for API version bump :+1: